src/client/src/orangehrmDashboardPlugin/components/EmployeesOnLeaveWidget.vue

```diff
@@ -0,0 +1,142 @@
+<!--
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+ -->
+
+<template>
+  <base-widget
+    icon="leavelist"
+    icon-type="svg"
+    class="emp-leave-chart"
+    :empty="isEmpty"
+    :empty-text="emptyText"
+    :loading="isLoading"
+    :title="$t('general.employees_on_leave_today')"
+  >
+    <div v-for="leave in leaveList" :key="leave" class="orangehrm-leave-card">
+      <div class="orangehrm-leave-card-profile-image">
+        <img
+          alt="profile picture"
+          class="employee-image"
+          :src="`../pim/viewPhoto/empNumber/${leave.empNumber}`"
```

Comment:
is this url okay for the component, since not a page. 
it's okay since we are using only one place for the moment

Reply:
using this same endpoint few places. can refactor the url to a computed if needed.